install/kubernetes/Makefile

```diff
@@ -18,20 +17,23 @@ CILIUM_CHART_REGEX := '\([vV]ersion:\) '$(VERSION_REGEX)
 CILIUM_TAG_REGEX := '\(tag:\) \(v'$(VERSION_REGEX)'\|latest\)'
 CILIUM_PULLPOLICY_REGEX := '\([pP]ullPolicy:\) .*'
 EXPERIMENTAL_OPTIONS := \
-    --set global.hubble.enabled=true \
-    --set global.hubble.listenAddress=":4244" \
-    --set global.hubble.tls.enabled=false \
-    --set global.hubble.metrics.enabled="{dns,drop,tcp,flow,port-distribution,icmp,http}" \
-    --set global.hubble.relay.enabled=true \
-    --set global.hubble.ui.enabled=true
+    --set hubble.enabled=true \
+    --set hubble.listenAddress=":4244" \
+    --set hubble.tls.enabled=false \
+    --set hubble.metrics.enabled="{dns,drop,tcp,flow,port-distribution,icmp,http}" \
+    --set hubble-relay.enabled=true \
+    --set hubble-relay.tls.enabled=false \
+    --set hubble-ui.enabled=true
+OPTS := \
+    --set hubble.tls.enabled=false
```

Comment:
It's my understanding that hubble has been promoted to on-by-default for Cilium 1.9. This is required to turn off hubble's TLS socket and avoid the quick-install failing diff tests.

Reply:
Hubble TLS cert generation should only happen if Hubble is configured to listen on `:4244` and Hubble should only be configured that way if Hubble Relay is enabled. When Hubble is in local mode (ie: unix domain socket only), TLS settings should be ignored as TLS is and will never enabled on the unix domain socket.

Reply:
> Assume that flattening hem charts is discussed before, I like it 💯
> 
> Couple of comments I have for first round review, will take a closer look later.
> 
> Edit: seems like prometheus-crds got removed in this PR, is it intentional ?

yes, removing the CRD generation is intentional because it duplicates a CRD created by Prometheus and  is just an integration point, we do not own the prometheus service nor should we create CRDs on behalf of Prometheus.